include/avif/avif.h

```diff
@@ -1160,6 +1160,9 @@ typedef struct avifDecoder
     // Can be useful to decode the gain map image only.
     avifBool ignoreColorAndAlpha;
 #endif
+
+    // This is true when avifDecoderParse() detects an animation track in the image.
```

Comment:
I suggest adding the following:
```c
// Note that a still image item may be present next to the animation track.
```
I remember `avifenc` generating `meta` and `moov` boxes by default, so I assume this is true.

Reply:
Yes, an AVIF animated image is always required to have a primary image item (i.e.) it should also be possible to decode the animated image as a still image file.

I have updated the comment to reflect this.